### ISSUE TITLE
[HANDS-ON BUG] Unit 1 problem with LFS files pushing model to the hub

### ISSUE THREAD

**Issue by AlvLC** [huggingface/deep-rl-class#460]
# Bug description

Hi there, I've strictly followed the notebook (checking the solutions for each cell) from the hands-on exercise in unit 1 and I always encounter this problem when I try to push the model to the hub: 

![image](https://github.com/huggingface/deep-rl-class/assets/91500253/50138297-ab63-4f52-8059-7ba329987de9)

Basically, the repository is created and the login to HF is fine but it gets stuck indefinitely waiting for the LSF files to be uploaded. I've tried to install LSF again and searched for information about this issue but I can't seem to find what is going on. I'll share the notebook bellow in case that anyone can help and there's anything that I oversaw but I find it weird.

# Material

 - Colab notebook: https://colab.research.google.com/drive/1xWTD_M-1j5ZdlcYQs2SvwG-XubwLVGcj?usp=sharing

**Comment by sardorboboshov** [comment]
I am having same problem

**Comment by magixn** [comment]
I found that 4 LFS in the official [model](https://huggingface.co/ThomasSimonini/ppo-LunarLander-v2) are all smaller than 200KB. I wonder it may be not a network issue. I also cannot visit the leaderboard after Runtime error during upload with 

> It appears that one or more of your files contain valid Hugging Face secrets, such as tokens or API keys. It is important to keep your secrets private and not expose them in code that is publicly accessible.

I think I do not do anything else but fill the TODO and run the notebook.

**Comment by andlock97** [comment]
I'm also having the same problem

**Comment by ferdmartin** [comment]
I am having this issue as well.

**Comment by JamesMurrayCoder** [comment]
I am also having this issue

**Comment by Drakrig** [comment]
I've managed to push it successfully by passing token directly to `package_to_hub`  function like this:

`package_to_hub(model=model, # Our trained model
               model_name=model_name,
               model_architecture=model_architecture, 
               env_id=env_id, 
               eval_env=eval_env, 
               repo_id=repo_id, 
               token=TOKEN_AS_STRING,
               commit_message=commit_message)`

Looks like all this scheme with `notebook_login()` and `git config --global credential.helper store` doesn't work as intended in Colab for now.

**Comment by ferdmartin** [comment]
@Drakrig Thank you so much. Your approach worked for me.

**Comment by AlvLC** [comment]
Also worked for me. I've had other similar issues trying to use some HF classes with the usual login instead of just passing the token. Thanks you @Drakrig !!!

**Comment by Wauplin** [comment]
Hi everyone, sorry for the inconvenience on this topic. The bug has been identified and fixed in `huggingface_hub`. More details in https://github.com/huggingface/huggingface_hub/pull/1953#issuecomment-1878364894.

**TL;DR:** in the meantime you can fix the problem by running `!pip install -U huggingface_hub` before running anything else in your colab.
